emuu-client/src/components/videoPage/videoPage.js

```diff
@@ -0,0 +1,172 @@
+              checkedIcon={<Favorite />}
+              name="Like"
+              id="Like"
+              checked={checked}
+              onChange={async (e) => {
+                setChecked(!checked);
+                const collectionRef = collection(db, "Videos");
+                const queryData = await query(
+                  collectionRef,
+                  where("VideoUrl", "==", video.VideoUrl)
+                );
+                const _doc = await getDocs(queryData);
+                let id = "";
+                _doc.forEach((doc) => (id = doc.id));
+                const videoRef = doc(db, "Videos", id);
+                if (e.target.checked) {
+                  await updateDoc(videoRef, { Likes: increment(1) });
+                  await updateDoc(videoRef, {
+                    usersThatLiked: arrayUnion(displayName),
+                  });
+                } else {
+                  await updateDoc(videoRef, { Likes: increment(-1) });
+                  await updateDoc(videoRef, {
+                    usersThatLiked: arrayRemove(displayName),
+                  });
+                }
+                setVideo((await getDoc(videoRef)).data());
+              }}
+            />
+          }
+          label="Like"
+        />
+      </p>
+      <p>Posted By: {video.Username} on </p>
+      <p>Game Tag: {video.GameTag}</p>
+      <TextField
+        id="standard-textarea"
+        label="Enter a comment"
+        placeholder=""
+        multiline
+        variant="standard"
+        value={comment}
+        onChange={handleComments}
+      />{" "}
+      <p></p>
+      <button
+        class="btn btn-primary"
+        type="submit"
+        onClick={async () => {
+          setComment("");
+          const collectionRef = collection(db, "Videos");
+          const queryData = await query(
+            collectionRef,
+            where("VideoUrl", "==", video.VideoUrl)
+          );
+          const _doc = await getDocs(queryData);
+          let id = "";
+          _doc.forEach((doc) => (id = doc.id));
+          const videoRef = doc(db, "Videos", id);
+          await updateDoc(videoRef, {
+            Comments: arrayUnion({
+              text: comment,
+              postedBy: displayName,
+              date: new Date().toLocaleDateString(),
+            }),
+          });
+          setVideo((await getDoc(videoRef)).data());
+        }}
+      >
+        Submit
+      </button>
+      {video.Comments && (
+        <div className="comments">
+          <h2>Comments</h2>
+          {video.Comments.map((comment) => (
+            <div className="comment">
+              <p>
+                <h3>{comment.text}</h3>
+              </p>
+              <p>
+                <h5>
+                  Posted by {comment.postedBy} on {comment.date}
+                </h5>
+              </p>
+            </div>
+          ))}
+        </div>
+      )}
+    </div>
+  );
+}
+
+export default Video;
```

Comment:
Overall good work on the videos page. Just need proper styling. I think if we have visible containers for the description and the comment section it would look nicer, and having the videos margin be according to screen size.